resources/META-INF/plugin.xml

```diff
@@ -1,7 +1,7 @@
 <idea-plugin>
   <id>com.github.johnthagen.cppcheck</id>
   <name>cppcheck</name>
-  <version>1.6.4</version>
+  <version>1.6.5</version>
```

Comment:
Can we save the version bump until after I make the 1.6.5 release?

That way I can tag 1.6.4 from `master` without the version bumped and then after that we can bump to 1.6.5

Reply:
That's why I made this induvial commits. If you merge this with a merge commit you preserve the commits and can set the tag on d1e6bae.

Reply:
Ah, okay. I'm used to having all PRs squash by default.

Okay, I can make the release later today.

Reply:
Yeah - I am also used to squashes since the merge commits are annoying. But on a very small repos with basically a single developer I usually go with a dev branch with direct commits. So this is a hybrid approach.